# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/e17/imlib2/makefile.am.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ru imlib2-2002-08-18_/src/Makefile.am imlib2-2002-08-18/src/Makefile.am
--- imlib2-2002-08-18_/src/Makefile.am	2001-08-10 08:35:05.000000000 +0000
+++ imlib2-2002-08-18/src/Makefile.am	2002-11-21 12:25:37.000000000 +0000
@@ -13,6 +13,8 @@
 MMX_SRCS             = asm_blend.S asm_rgba.S asm_scale.S asm_rotate.S \
 		       asm_blend_cmod.S
 MMX_OBJS             = $(MMX_SRCS:.S=.lo)
+CCAS = $(AS)
+CCASFLAGS = $(ASFLAGS)
 
 lib_LTLIBRARIES      = libImlib2.la
 include_HEADERS      = Imlib2.h
